usagov/vote-gov-drupal
# VOTE-1117 Remove landing page (block) component field and dependencies

Set new homepage in site settings
Uninstall inline entity form module
Remove unused block types
Remove unused templates

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1117

## Description (optional)
This completes step 8 in the ticket to remove deprecated code for the landing page blocks

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. You may be required to delete the block content for basic, accordion group, registration tool, blocks. This can be done from here: http://vote-gov.lndo.site/admin/structure/block/block-content

### Post-deploy steps
1. run `lando retune`

### Testing steps
1. If you do not have a node 63 the homepage will probably be a page not found. When deployed to dev it will go to the correct location: https://vote-drupal-dev.app.cloud.gov/node/63
2. This is removing custom block types and the page components field from the landing page. You can confirm that accordion group and registration tool block types have been removed. Basic block type will remain and be a general mulit-purpose block type for future use. You should see the old page components field has been replaced with paragraphs: http://vote-gov.lndo.site/node/add/landing
